src/HOME4/home4_2.sql

```diff
@@ -0,0 +1,122 @@
+drop database if exists car;
+create database car;
+use car;
+
+
+CREATE TABLE owners(
+id int not null primary key auto_increment,
+name_of_owners varchar(30) not null,
+surname_of_owners varchar(30) not null,
+telephone integer not null,
+email varchar(30),
+town_id int not null,
+country_id int not null
+);
+
+insert into owners (name_of_owners, surname_of_owners, telephone, email,town_id,country_id) values
+('Микола', 'Петренко', 968235091, 'email@mail.com',1,3),
+('Omar', 'Nymiz', 487552901, 'ny@gmail.com',3,1),
+('Richard', 'Grei', 318375420,  'grei@gmail.com',2,1);
+
+CREATE TABLE employees(
```

Comment:
I can't understand why do we have `employees ` and `owners`. They both contain persons.
Better will be if we merge this table with `owners`  and create some field(or table) to identify if the user is owner whether employee.